src/main/java/com/notably/view/blockcontent/BlockContentDisplayView.java

```diff
@@ -21,47 +28,126 @@
     private final Model model;
 
     @FXML
-    private WebView blockContentDisplay;
+    private VBox blockContentDisplay;
+
+    @FXML
+    private HBox blockTitlePathContainer;
+
+    @FXML
+    private Label blockTitleDisplay;
+
+    @FXML
+    private Pane separator;
+
+    @FXML
+    private Label blockPathDisplay;
+
+    @FXML
+    private WebView blockBodyDisplay;
 
     public BlockContentDisplayView(Model model) {
         super(FXML);
-
         Objects.requireNonNull(model);
-
         this.model = model;
-
         setup();
     }
 
     /**
-     * Sets up the view's initial data and wire up all required change listeners.
+     * Sets up the view's initial data and wires up all required change listeners.
      */
     private void setup() {
-        // Initialize content display
-        reload(model);
+        setSettings();
+        updateBlockContentDisplay(model);
+        setChangeListeners();
+    }
 
-        // Setup change listeners
-        model.currentlyOpenPathProperty().addListener(observable -> reload(model));
-        model.getBlockTree().getRootBlock().getTreeItem()
-                .addEventHandler(TreeItem.treeNotificationEvent(), event -> reload(model));
+    private void setSettings() {
+        blockPathDisplay.setTextAlignment(TextAlignment.RIGHT);
+        blockTitlePathContainer.setMinWidth(50);
+        blockTitlePathContainer.setMinWidth(0);
+        blockTitlePathContainer.setPrefWidth(1);
+        blockTitlePathContainer.setAlignment(Pos.CENTER_LEFT);
+        blockTitlePathContainer.setHgrow(blockPathDisplay, Priority.SOMETIMES);
+        blockTitlePathContainer.setHgrow(separator, Priority.SOMETIMES);
+        blockTitlePathContainer.setHgrow(blockTitleDisplay, Priority.ALWAYS);
```

Comment:
Nitpicking here but I believe the recommended way of calling static methods is `<ClassName>.<static method name>`. For that reason, this should be `HBox.setHgrow`. You can actually set all these parameters directly in the `fxml` files too!

Reply:
I've converted the inline stylings to fxml. Thanks.